internal/integration/provision/upgrade.go

```diff
@@ -394,6 +394,10 @@ func (suite *UpgradeSuite) setupCluster() {
 		)
 	}
 
```

Comment:
I had to add this (disabling kexec in upgrade tests), as otherwise tests are broken as VMs get stuck in random places. not sure what we can do about it, whether it's QEMU only, or that is a real issue

Reply:
created #6443 to track this